skbio/io/format/genbank.py

```diff
@@ -0,0 +1,865 @@
+...       'DEFINITION  Chain A, Structure Of A Mutant Class-I Preq1.\\n',
+...       'ACCESSION   3K1V_A\\n',
+...       'VERSION     3K1V_A  GI:260656459\\n',
+...       'KEYWORDS    .\\n',
+...       'SOURCE      synthetic construct\\n',
+...       '  ORGANISM  synthetic construct\\n',
+...       '            other sequences; artificial sequences.\\n',
+...       'REFERENCE   1  (bases 1 to 34)\\n',
+...       "  AUTHORS   Klein,D.J., Edwards,T.E. and Ferre-D'Amare,A.R.\\n",
+...       '  TITLE     Cocrystal structure of a class I preQ1 riboswitch\\n',
+...       '  JOURNAL   Nat. Struct. Mol. Biol. 16 (3), 343-344 (2009)\\n',
+...       '   PUBMED   19234468\\n',
+...       'COMMENT     SEQRES.\\n',
+...       'FEATURES             Location/Qualifiers\\n',
+...       '     source          1..34\\n',
+...       '                     /organism="synthetic construct"\\n',
+...       '                     /mol_type="other RNA"\\n',
+...       '                     /db_xref="taxon:32630"\\n',
+...       'ORIGIN\\n',
+...       '        1 agaggttcta gcacatccct ctataaaaaa ctaa\\n',
+...       '//\\n']
+
+Now we can read it as ``DNA`` object:
+
+>>> from skbio import DNA, RNA, Sequence
+>>> dna_seq = DNA.read(gb)
+>>> dna_seq
+DNA
+-----------------------------------------------------------------
+Metadata:
+    'ACCESSION': '3K1V_A'
+    'COMMENT': 'SEQRES.'
+    'DEFINITION': 'Chain A, Structure Of A Mutant Class-I Preq1.'
+    'FEATURES': <class 'list'>
+    'KEYWORDS': '.'
+    'LOCUS': <class 'dict'>
+    'REFERENCE': <class 'list'>
+    'SOURCE': <class 'dict'>
+    'VERSION': '3K1V_A  GI:260656459'
+Positional metadata:
+    0: <dtype: bool>
+Stats:
+    length: 34
+    has gaps: False
+    has degenerates: False
+    has non-degenerates: True
+    GC-content: 35.29%
+-----------------------------------------------------------------
+0 AGAGGTTCTA GCACATCCCT CTATAAAAAA CTAA
+
+
+Since this is a riboswitch molecule, we may want to read it as ``RNA``.
+As the GenBank file usually have ``'t'`` instead of ``'u'`` in the
+sequence, we can not read it as ``RNA`` directly without conversion.
+We can read it as ``DNA`` and then ``transcribe`` it into ``RNA``:
+
+>>> rna_seq = dna_seq.transcribe()
+>>> rna_seq
+RNA
+-----------------------------------------------------------------
+Metadata:
+    'ACCESSION': '3K1V_A'
+    'COMMENT': 'SEQRES.'
+    'DEFINITION': 'Chain A, Structure Of A Mutant Class-I Preq1.'
+    'FEATURES': <class 'list'>
+    'KEYWORDS': '.'
+    'LOCUS': <class 'dict'>
+    'REFERENCE': <class 'list'>
+    'SOURCE': <class 'dict'>
+    'VERSION': '3K1V_A  GI:260656459'
+Positional metadata:
+    0: <dtype: bool>
+Stats:
+    length: 34
+    has gaps: False
+    has degenerates: False
+    has non-degenerates: True
+    GC-content: 35.29%
+-----------------------------------------------------------------
+0 AGAGGUUCUA GCACAUCCCU CUAUAAAAAA CUAA
+
+
+Or we can convert ``'t'`` to ``'u'`` in the sequence and then read it
+as ``RNA``:
```

Comment:
I see now. I think the conversion should be automatically handled as most (hopefully all) GenBank files will use a `t` in the context of `RNA`. In other words, both a file containing `u` and `t` may be read into `RNA`